android/assets/jsons/translations/Traditional_Chinese.properties

```
@@ -7214,32 +7214,32 @@ Global =
  # Requires translation!
 Nation = 
  # Requires translation!
-Era = 
+Era = 紀元
  # Requires translation!
-Tech = 
+Tech = 科技
  # Requires translation!
-Policy = 
+Policy = 政策
  # Requires translation!
 FounderBelief = 
  # Requires translation!
 FollowerBelief = 
  # Requires translation!
-Building = 
+Building = 建造中
```

```suggestion
Building = 建筑
```
I wonder why I missed this mistake. The `building` should be easy to find in game, and you can find them in the cityscreen.